cluster/examples/kubernetes/ceph/common.yaml

```
@@ -1082,6 +1082,15 @@ rules:
   - apiGroups: [""]
     resources: ["configmaps"]
     verbs: ["get"]
+  - apiGroups: ["replication.storage.openshift.io"]
+    resources: ["volumereplications"]
+    verbs: ["create", "delete", "get", "list",  "patch",   "update", "watch"]
```

```suggestion
    verbs: ["create", "delete", "get", "list", "patch", "update", "watch"]
```

see yamllint is failing due to this.